typescript/commands/error_info.py

```diff
@@ -21,7 +21,11 @@ def run(self, text):
             if region.contains(pt):
                 error_text = text
                 break
+
         if len(error_text) > 0:
-            self.view.set_status("typescript_error", error_text)
-        else:
-            self.view.erase_status("typescript_error")
\ No newline at end of file
+            if PHANTOM_SUPPORT:
```

Comment:
If `show_errors_inline` is set to `false`, do phantoms still appear? As this is an `if/else`, that would mean no error info. Is there a downside to always updating the status bar anyway?